CHANGES.md

```diff
@@ -6,6 +6,7 @@ Release Notes.
 ------------------
 
 * Upgrade nats plugin to support 2.16.5
+* [pulsar] Handle the SendCallback class method signature change.
```

Comment:
```suggestion
* Support newer Pulsar versions to adopt SendCallback class method signature change.
```

I can't see which versions will include this. The PR you refer to doesn't have the version information.

Reply:
The code implementation PR: https://github.com/apache/pulsar/pull/23196.
It will available in the next release.

Reply:
We can't accept this PR until it is released and verified through CI tests.

We have to hold this until then.